sympy/ntheory/primetest.py

```diff
@@ -488,9 +487,19 @@ def isprime(n):
     >>> from sympy.ntheory import isprime
     >>> isprime(13)
     True
+    >>> isprime(13.0)  # limited precision
+    False
     >>> isprime(15)
     False
 
+    Notes
+    =====
+
+    This routine is not meant to deal with expressions which may
```

Comment:
I think that removing the word not in the line above makes more sense.

Reply:
Please see modified docstring.

Reply:
I think that "This routine is not meant to deal with expressions which may represent a prime number" is misleading since the opposite is true. How about "This routine is intended to test whether a number definitely represents a prime number"?